auto_update_tests.py

```diff
@@ -18,15 +18,6 @@
     actual, err = subprocess.Popen([os.path.join('bin', 'wasm2asm'), os.path.join('test', wasm)], stdout=subprocess.PIPE, stderr=subprocess.PIPE).communicate()
     open(os.path.join('test', asm), 'w').write(actual)
 
-for t in sorted(os.listdir('test')):
-  if t.endswith('.wast') and not t.startswith('spec'):
-    print '..', t
-    t = os.path.join('test', t)
-    actual, err = subprocess.Popen([os.path.join('bin', 'binaryen-shell'), t, '--print-before'], stdout=subprocess.PIPE, stderr=subprocess.PIPE).communicate()
-    actual = actual.replace('printing before:\n', '')
-
-    open(t, 'w').write(actual)
```

Comment:
yes, see the commit. this was doing a wast=>wast transform. But it's impossible to do when the spec changes - the before state will not parse.